deviceplugin/usb.go

```diff
@@ -86,32 +110,32 @@ func queryUSBDeviceCharacteristicsByDirectory(dir os.DirEntry) (result *usbDevic
 		// There shouldn't be any raw files in this directory, but just in case.
 		return result, fmt.Errorf("not a directory")
 	}
+	fqPath := filepath.Join(usbDevicesDir, dir.Name())
 	// Try to find the vendor ID file inside this device - this is a good indication that we're dealing with a device, not a bus.
-	vnd, err := os.ReadFile(dir.Name() + "/" + usbDevicesDirVendorIDFile)
+	vnd, err := readFileToUint16(filepath.Join(fqPath, "/", usbDevicesDirVendorIDFile))
```

Comment:
No need to put a slash in there yourself, that's part of the lovely magic of filpath.Join!
```suggestion
	vnd, err := readFileToUint16(filepath.Join(fqPath, usbDevicesDirVendorIDFile))
```